tools/cli/commands.js

```diff
@@ -1212,12 +1213,16 @@ Instructions for publishing your iOS app to App Store can be found at:
 https://guide.meteor.com/cordova.html#submitting-ios
 `, "utf8");
             } else if (platform === 'android') {
-              const apkPath = files.pathJoin(buildPath, 'build/outputs/apk',
-                options.debug ? 'android-debug.apk' : 'android-release-unsigned.apk');
+              const packageType = options.packageType || "bundle"
+              const packageExtension = packageType === 'bundle' ? 'aab' : 'apk';
+              const packageName = packageType === 'bundle' ? `app-release` : `app-release-unsigned`;
```

Comment:
Would be nice if we could get here the name of the app. Probably from `package.json`.